### ISSUE TITLE
Auto update of planner version to fabric8-ui is broken

### ISSUE THREAD

**Issue by nimishamukherjee** [openshiftio/openshift.io#2825]
After #2749 was fixed, we discovered another issue.

Auto update of planner version to fabric8-ui is broken.

Error on fabric8-npm-dependencies:
https://jenkins.cd.test.fabric8.io/blue/organizations/jenkins/fabric8-ui%2Ffabric8-npm-dependencies/detail/master/178/pipeline

**Comment by pranavgore09** [comment]
@jaseemabid I know you are aware of it but just mentioning here for others that changes made for planner's [workaround](https://github.com/fabric8io/fabric8-pipeline-library/pull/395/files) are needed [here](https://sourcegraph.com/github.com/fabric8io/fabric8-pipeline-library/-/blob/vars/npmUpdateOrganisationDependencies.groovy#L82) as well

**Comment by jaseemabid** [comment]
@pranavgore09 Noted. I'll patch npmUpdateOrganisationDependencies.groovy as well.

**Comment by joshuawilson** [comment]
Can you fix all the build scripts for each of the repos in https://github.com/fabric8-ui

**Comment by rupalibehera** [comment]
Here is the PR https://github.com/fabric8io/fabric8-pipeline-library/pull/396 from @jaseemabid , need to test it and merge it.

**Comment by joshuawilson** [comment]
I thought this was NOT going to get merged due to the risk of breaking upstream external projects.

**Comment by joshuawilson** [comment]
This has started working again, magically. 
This also should not be a P0 as the user never sees this.

**Comment by piyush1594** [comment]
The release of Fabric8 Kubernetes Model is still facing the issue. Reopening it.

**Comment by joshuawilson** [comment]
I didn't know we had Fabric8 Kubernetes Model auto releasing to fabric8-ui.

**Comment by piyush1594** [comment]
Here is the link to the job I just started after your comment @joshuawilson https://jenkins.cd.test.fabric8.io/job/fabric8io/job/kubernetes-model/job/master/20/console

**Comment by pradeepto** [comment]
@joshuawilson This is p0 because it blocks releases of few bits - including the jenkins-openshift-sync plugin which is required by Idler.

**Comment by joshuawilson** [comment]
@pradeepto it looks like you are raising an issue not related to the title.  The cause and fix may be related. But does this block Planner (or anything else) from pushing to fabric8-ui?

**Comment by jaseemabid** [comment]
I probably fixed this as part of https://github.com/openshiftio/openshift.io/issues/3324. Can we reproduce this again?

**Comment by piyush1594** [comment]
@jaseemabid We checked for kubernetes-model and it worked.
